portfolio/src/main/webapp/index.html

```diff
@@ -29,55 +28,128 @@ <h3>About Me</h3>
+          I changed my mind about languages. No longer did they seem dull, confusing
+          and monotonous to learn! I like to use lots of different techniques, and I
+          especially enjoy reading foreign books, and using Duolingo 
+          <a href="http://duome.eu/larmstrong24" target="_blank">(see my stats!)</a>
+        </p>
+        <!--Flag Images as Overlays-->
+        <div class="flag left-content">
+          <div class="flag-overlay"></div>
+            <img src= "/images/french-flag.png" alt="French Flag" width=100px height=67px margin-right=0>
+            <div class="flag-details fadeIn-top">
+            <h3>French</h3>
+          </div>
+        </div>
+        <div class="flag left-content">
+          <div class="flag-overlay"></div>
+            <img src= "/images/spanish-flag.png" alt="Spanish Flag" width=100px height=67px margin-right=0>
+            <div class="flag-details fadeIn-top">
+            <h3>Spanish</h3>
+          </div>
+        </div>
+        <div class="flag left-content">
+          <div class="flag-overlay"></div>
+            <img src= "/images/german-flag.png" alt="German Flag" width=100px height=67px margin-right=0>
+            <div class="flag-details fadeIn-top">
+            <h3>German</h3>
+          </div>
+        </div>
+        <div class="flag left-content">
+          <div class="flag-overlay"></div>
+            <img src= "/images/italian-flag.png" alt="Italian Flag" width=100px height=67px margin-right=0>
+            <div class="flag-details fadeIn-top">
+            <h3>Italian</h3>
+          </div>
+        </div>
+        <div class="flag left-content">
+          <div class="flag-overlay"></div>
+            <img src= "/images/greek-flag.png" alt="Greek Flag" width=100px height=67px margin-right=0>
+            <div class="flag-details fadeIn-top">
+            <h3>Greek</h3>
+          </div>
+        </div>
+        <div class="flag left-content">
+          <div class="flag-overlay"></div>
+            <img src= "/images/dutch-flag.jpg" alt="Dutch Flag" width=100px height=67px margin-right=0>
+            <div class="flag-details fadeIn-top">
+            <h3>Dutch</h3>
+          </div>
+        </div>
+        <div class="flag left-content">
+          <div class="flag-overlay"></div>
+            <img src= "/images/swedish-flag.png" alt="Swedish Flag" width=100px height=67px margin-right=0>
+            <div class="flag-details fadeIn-top">
+            <h3>Swedish</h3>
+          </div>
+        </div>
+        <img class="extra-border-img" src= "/images/arabic.png" alt="Arabic Flag" width=112px height=75px>
+      </section>
+      <section class="center-content">
+        <div class="buttonText">
+          <a href="https://www.linkedin.com/in/lornafarmstrong" target="_blank">
+          <button class="hyperlinked-button buttonBlue">LinkedIn</button></a>
+          <a href="https://www.https://github.com/lornaarmstrong" target="_blank">
```

Comment:
please fix this typo before submitting :)

Reply:
Will do, thanks!